### PR TITLE
InfoAndCtaContainer: SDG Cards Section

### PR DESCRIPTION
**Pull request by prachigarg19**
This PR adds a new component at the bottom of public profile which lists all the SDG cards

image

### CODE REVIEW

**Review by mariahosfeld — DISMISSED** [review]
- [x] Once the last SDG has been reached, either the right arrow of the slider should deactivate or it should start again with the first goal
- [x] If possible the SDGs should be closer together (check Design for reference)

**Comment by mohitb35** [comment]
Remove the unnecessary sdgs.

File: public/static/locales/en/profile.json [hunk 1]
```diff
@@ -108,6 +108,28 @@
...
+        "industryInfrastructureAndInnovation": "INDUSTRY, INNOVATION AND INFRASTRUCTURE", 
+        "reducedInequalities": "recent inequalities",
+        "sustainableCities": "SUSTAINABLE CITIES AND COMMUNITIES",
+        "responsibleConsumptionAndProduction": "RESPONSIBLE CONSUMPTION AND PRODUCTION",
+        "climateAction": "climate action",
+        "lifeBelowWater": "life below water",
+        "lifeOnLand": "life on land",
+        "peace": "PEACE, JUSTICE AND STRONG INSTITUTIONS",
+        "partnership": "PARTNERSHIPS FOR THE GOALS"
+      }
```
Comment (mohitb35) [inline public/static/locales/en/profile.json:132]: I'd suggest entering the translation strings in Sentence case. We are capitalizing the text using text-transform in any case.

File: src/features/common/CustomControlledCarousel/index.tsx
Comment (mohitb35) [inline src/features/common/CustomControlledCarousel/index.tsx:1]: 1. Resolve the TS warnings in this file, including for the props.
2. `SampleArrow` --> perhaps `CarouselArrow` would be a better name

File: src/features/user/MFV2/InfoAndCTA/SDGCardList/SingleSDGCard.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+interface Props {
+  title: string;
+  icon: ReactElement;
+  index: number;
+  color: string;
+}
+
+const SingleSDGCard = ({ title, icon, index, color }: Props) => {
+  const t = useTranslations('Profile');
```
Comment (mohitb35) [inline src/features/user/MFV2/InfoAndCTA/SDGCardList/SingleSDGCard.tsx:13]: Translations are not used here.

File: src/features/user/MFV2/InfoAndCTA/SDGCardList/SingleSDGCard.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  return (
+    <div
+      className={styles.singleSDGCardContainer}
+      style={{ background: color }}
+    >
+      <div>
+        <span className={styles.singleCardIndex}>
+          {String(index + 1).padStart(2, '0')}
```
Comment (mohitb35) [inline src/features/user/MFV2/InfoAndCTA/SDGCardList/SingleSDGCard.tsx:21]: For readability, could you create a helper function for this?

File: src/features/user/MFV2/InfoAndCTA/SDGCardList/SDGCardsData.tsx
Comment (mohitb35) [inline src/features/user/MFV2/InfoAndCTA/SDGCardList/SDGCardsData.tsx:1]: Update file name to SDGElements to match the component name

File: src/features/user/MFV2/InfoAndCTA/SDGCardList/index.tsx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+        breakpoint: 380,
+        settings: {
+          centerMode: false,
+          slidesToShow: 1,
+        },
+      },
+    ],
+  };
+
+  const sdgCardsDataArray = SDGElements();
```
Comment (mohitb35) [inline src/features/user/MFV2/InfoAndCTA/SDGCardList/index.tsx:77]: Is this needed? We could just pass SDGElements as `carouselData`?

**Review by mohitb35 — CHANGES_REQUESTED** [review]
@prachigarg19 Please review my feedback

File: public/static/locales/en/profile.json (hunk 1 above)
Comment (prachigarg19) [reply public/static/locales/en/profile.json:132]: Not needed anymore

File: src/features/user/MFV2/InfoAndCTA/SDGCardList/SingleSDGCard.tsx (hunk 2 above)
Comment (prachigarg19) [reply src/features/user/MFV2/InfoAndCTA/SDGCardList/SingleSDGCard.tsx:13]: Not needed anymore

File: src/features/user/MFV2/InfoAndCTA/SDGCardList/SingleSDGCard.tsx (hunk 3 above)
Comment (prachigarg19) [reply src/features/user/MFV2/InfoAndCTA/SDGCardList/SingleSDGCard.tsx:21]: Not needed anymore

File: src/features/user/MFV2/InfoAndCTA/SDGCardList/SDGCardsData.tsx
Comment (prachigarg19) [reply src/features/user/MFV2/InfoAndCTA/SDGCardList/SDGCardsData.tsx:1]: Not needed anymore

File: src/features/user/MFV2/InfoAndCTA/SDGCardList/index.tsx (hunk 4 above)
Comment (prachigarg19) [reply src/features/user/MFV2/InfoAndCTA/SDGCardList/index.tsx:77]: Not needed anymore

File: public/static/locales/en/profile.json (hunk 1 above)
Comment (mohitb35) [reply public/static/locales/en/profile.json]: Please leave the relevant translation strings, and use them for alt text for the SDG images.

**Comment by mohitb35** [comment]
Images look a little hazy. Let's go with the original dimensions itself, so that this does not happen.

**Review by mohitb35 — APPROVED** [review]

**Pull request merged**